Automattic/woocommerce-payments
# refactor: remove DW gateway class from new settings

Fixes https://github.com/Automattic/woocommerce-payments/issues/1839

#### Changes proposed in this Pull Request









The remaining work for https://github.com/Automattic/woocommerce-payments/pull/1810 was to remove the dependency that has been introduced on the `Digital_Wallets_Payment_Gateway` - as discussed over Slack.

#### Testing instructions





- Ensure you have the "grouped settings" flag enabled in WCPay Dev Tools
- Go to your WCPay Home: http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout
- You can still check/uncheck DW and its locations and save the settings
- Disabling the "grouped settings" will show the same locations/settings on the old settings view - which will ensure that the settings are used on the frontend: https://d.pr/i/EFmNWY > https://d.pr/i/3umZbh

![2021-05-21 12 16 01](https://user-images.githubusercontent.com/273592/119174562-57d87e80-ba2e-11eb-9178-f146e4465d16.gif)


-------------------
